runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java

```diff
@@ -884,7 +896,7 @@ private int findIndex(MenuEntry[] entries, int limit, String option, String targ
 	{
 		if (strict)
 		{
-			List<Integer> indexes = optionIndexes.get(option);
+			List<Integer> indexes = optionIndexes.get(option.toLowerCase());
```

Comment:
Why is this changed?

Reply:
I changed this so that I could have the option strings in BuyMode and Sellmode have the same capitalization as they do in-game, rather than already being in lower case. Should I change BuyMode and Sellmode to just have the option be lowercase to start with?

Reply:
That would be best. :+1: